tests/cephfs/cephfs_scale/large_file_writes.py

```diff
@@ -0,0 +1,363 @@
+import json
+import os
+import random
+import string
+import threading
+import traceback
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from datetime import datetime
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils as FsUtilsV1
+from utility.log import Log
+
+log = Log(__name__)
+
+logging_thread = None
+stop_event = threading.Event()
+
+
+def run(ceph_cluster, **kw):
+    """
+    Main function to execute the test workflow.
+
+    This function sets up the clients, prepares the CephFS environment, creates subvolumes,
+    mounts them, runs IO operations, and collects logs and status information.
+
+    :param ceph_cluster: Ceph cluster object
+    :param kw: Additional keyword arguments for configuration
+    :return: 0 if successful, 1 if an error occurs
+    """
+    try:
+        fs_util_v1 = FsUtilsV1(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        fs_util_v1.prepare_clients(clients, build)
+        fs_util_v1.auth_list(clients)
+
+        default_fs = "cephfs"
+        subvolume_group_name = "subvolgroup1"
+        subvolume_name = "subvol"
+
+        client_count = config.get("num_of_clients", 0)
+        mount_type = config.get("mount_type", "fuse")
+
+        if client_count > len(clients):
+            log.error(
+                f"Clients required to perform the test is {client_count} but "
+                f"the conf file has only {len(clients)}"
+            )
+            return 1
+
+        fs_util_v1.enable_mds_logs(clients[0], default_fs)
+
+        mds_list = get_daemon_names(clients[0], "mds")
+        log.info(f"MDS Daemons are : {mds_list}")
+
+        cmd_list = [
+            "ceph crash ls",
+            "ceph fs status",
+            "ceph fs dump",
+            "ceph -s",
+        ]
+        # Append ceph tell commands for MDS daemons
+        mds_list = get_daemon_names(clients[0], "mds")
```

Comment:
It will filter all the MDS nodes, including those from other filesystems. Is it acceptable to collect data from all these nodes, 
If that is the intention you can ignore this comment

Reply:
But, the same MDSs will be used by other FSs also as standby daemons right. We won't have control on that.
I will consider this for enhancements later.
Thanks Amar.